server/utils/utils.ts

```diff
@@ -74,83 +74,80 @@ export const formatAssessmentData = (
   if (!assessment || !assessment.sanAssessmentData) {
     return { lowScoring: [], highScoring: [], other: [] }
   }
-  const all = Object.values(areas)
-    .map(area => {
-      let score
-      let linkedtoRoSH
-      let linkedtoReoffending
-      let subData: SubAreaData
-      let overallScore
-
-      if (Object.prototype.hasOwnProperty.call(crimNeeds, area.crimNeedsKey)) {
-        score = crimNeeds[area.crimNeedsKey][`${area.crimNeedsSubKey}OtherWeightedScore`]
-        linkedtoRoSH = crimNeeds[area.crimNeedsKey][`${area.crimNeedsSubKey}LinkedToHarm`] === 'YES'
-        linkedtoReoffending = crimNeeds[area.crimNeedsKey][`${area.crimNeedsSubKey}LinkedToReoffending`] === 'YES'
-        if (Number.isNaN(Number(score))) {
-          score = undefined
-        } else if (score > area.upperBound) {
-          score = area.upperBound
-        }
+  const all = Object.values(areas).map(area => {
+    let score
+    let linkedtoRoSH
+    let linkedtoReoffending
+    let subData: SubAreaData
+    let overallScore
+
+    if (Object.prototype.hasOwnProperty.call(crimNeeds, area.crimNeedsKey)) {
```

Comment:
Any reason for doing this using `Object.prototype.hasOwnProperty.call`? seems a bit odd school-y. 

Could do 
- `if (area.crimNeedsKey in crimNeeds)`
- `if (area.crimNeedsKey[area.crimNeedsKey])` (if the value can come out as false, add an undefined check)
- `if (Object.hasOwn(crimNeeds, area.crimNeedsKey)`

I can't imagine there'll be a prototype property collision with the the expected data on this object, so you're probably okay to do a less robust check like the first 2 examples

Reply:
I didn't write this line, so I don't know. This was a whitespace change. I can change it though.

Reply:
Fixed in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/260/commits/6be11af672980c9e4990cddececcf804bfd7efd8